index.html

```diff
@@ -19,7 +19,7 @@ <h3 id="heading" class="lead text-light mt-2 mb-5">Search for any show below!</h
     </header>
 
     <!-- Main Body -->
-    <main>
+    <main> <!-- Comment on bootstrap -->
```

Comment:
If you're using bootstrap, you should have a `class="container"` here so that the classes will all work properly.  I also noticed that you're using columns on initial page load but you're missing them when you update content on the page.

You may want to review how the [Bootstrap Grid](https://getbootstrap.com/docs/4.0/layout/grid/) system works.